source/stylesheets/base/_layout.scss

```diff
@@ -51,3 +51,12 @@ body {
     justify-content: space-around;
   }
 }
+
+@media only screen
+  and (max-device-width: 640px),
+  only screen and (max-device-width: 667px),
+  only screen and (max-width: 480px) {
```

Comment:
Aren't all of these saying the same thing, i.e. "if the screen is less that 667px then do something"?